client/post-editor/media-modal/image-editor/index.jsx

```diff
@@ -126,18 +139,41 @@ const MediaModalImageEditor = React.createClass( {
 		return ! transfer.types || -1 !== Array.prototype.indexOf.call( transfer.types, 'Files' );
 	},
 
+	onLoadCanvasError() {
+		this.setState( { canvasError: this.translate( 'We are unable to edit this image.' ) } );
+		this.timerId = setTimeout( this.props.onImageEditorCancel, 5000 );
+	},
+
+	renderError() {
+		return (
+			<Notice
+				status="is-error"
+				showDismiss={ true }
+				text={ this.state.canvasError }
+				isCompact={ false }
+				onDismissClick={ this.props.onImageEditorCancel } 	/>
+		);
+	},
+
 	render() {
 		return (
-			<div className="editor-media-modal-image-editor">
-				<figure>
-					<div className="editor-media-modal-image-editor__content editor-media-modal__content" >
-						<EditCanvas ref="editCanvas" />
-						<EditToolbar />
-						<EditButtons
-							onCancel={ this.props.onImageEditorCancel }
-							onDone={ this.onDone } />
-					</div>
-				</figure>
+			<div>
```

Comment:
The outermost div should have the component class name. It looks like we're not matching the naming guidelines at the moment, but let's leave that for another PR.